src/mbgl/renderer/layers/render_fill_extrusion_layer.cpp

```diff
@@ -293,10 +293,10 @@ void RenderFillExtrusionLayer::update(gfx::ShaderRegistry& shaders,
     }
 
     if (!fillExtrusionGroup) {
-        fillExtrusionGroup = shaders.getShaderGroup(std::string(FillExtrusionShaderName));
+        fillExtrusionGroup = shaders.getShaderGroup(MLN_STRINGIZE(FillExtrusionShader));
```

Comment:
Why not

```suggestion
        fillExtrusionGroup = shaders.getShaderGroup("FillExtrusionShader"));
```

?

And in a few other places as well.

Reply:
My hope was to get compile-time validation, but haven't been able to achieve it.